kong/pdk/request.lua

```diff
@@ -370,18 +371,49 @@ local function new(self)
   end
 
 
+  ---
+  -- Returns the normalized path component of the request's URL. The return
+  -- value is the same as `kong.request.get_raw_path()` but normalized according
+  -- to RFC 3986 section 6:
+  --
+  -- * Percent-encoded values of unreserved characters are decoded (`%20`
+  --   becomes ` `).
+  -- * Percent-encoded values of reserved characters have their hexidecimal
+  --   value uppercased (`%2f` becomes `%2F`).
+  -- * Relative path elements (`/.` and `/..`) are dereferenced.
+  -- * Duplicate slashes are consolidated (`//` becomes `/`).
+  --
+  -- @function kong.request.get_path
+  -- @phases rewrite, access, header_filter, response, body_filter, log, admin_api
+  -- @tparam[opt=true] boolean merge_slashes consolidate duplicate slashes in the path
```

Comment:
This does not appears to be on the PDK, and is probably unnecessary. We can just remove this line.
```suggestion
```